packages/discord.js/src/structures/interfaces/InteractionResponses.js

```diff
@@ -248,6 +249,38 @@ class InteractionResponses {
     this.replied = true;
   }
 
+  /**
+   * An object containing the same properties as CollectorOptions, but a few less:
```

Comment:
Might as well link to this for those curious about what `CollectorOptions` is.

```suggestion
   * An object containing the same properties as {@link CollectorOptions}, but a few less:
```